apps/vanilla/src/scripts/renderFilterByType.ts

```diff
@@ -0,0 +1,34 @@
+import { Type } from '@js-camp/core/enum';
+import { PaginationOptions } from '@js-camp/core/models/paginationOptions';
+import { setDefaultSelected } from '@js-camp/core/utils';
+
+import { FILTER_TYPE_OPTIONS } from '../constants';
+import { KEY_TYPE } from '../constants/key';
+import { getValueFromLocalStorage, setValueToLocalStorage } from '../service/localStorage';
+
+import { renderListAndPaginationToUI } from './renderPagination';
+
+const selectType = document.querySelector<HTMLSelectElement>('.filter__item_select-type');
+
+/** Render filter by type.
+ * @param options Pagination options.
+ */
```

Comment:
Please format multiline jsdoc properly, first and last lines shouldn't have text.
```suggestion
/**
 * Render filter by type.
 * @param options Pagination options.
 */
```